### ISSUE TITLE
Key & mouse binding autocompletion

### ISSUE THREAD

**Issue by hsandt** [slynch8/10x#2616]
Coming from https://github.com/slynch8/10x/issues/1966 opening its own issue.

Sometimes I'm not sure of an exact way to write an input name in key bindings. It's pretty easy to find examples of modifier keys like Control in neighbouring entries, but for Mouse buttons (see https://github.com/slynch8/10x/issues/2615) or keys used less often (PrintScreen? Backspace?) it's not very obvious.

They should all be described with their exact expected names in https://www.10xeditor.com/10xDocumentation/KeyBindings.htm , but autocompletion while editing the key bindings would be even better.

**Comment by slynch8** [comment]
Good idea. I will add autocomplete and add it to the docs. For now you can use the Menu Menu -> Log Keys to see what a key is called.
